languages/cpp/README.md

```diff
@@ -51,7 +52,7 @@ ProjectResponse projectResponseGet = bitwardenClient.getProject(projectId);
 
 ```c++
 boost::uuids::uuid projectId = boost::uuids::string_generator()(projectResponseCreate.get_id());
```

Comment:
⛏️ Why not extract`boost::uuids::uuid projectId` to be part of (at the end) `### Create new project` like we do in other language wrapper readme's ?

Reply:
Addressed in 7ab21345d0d91c7dd4a152ef7b33da507f2bae6b.